src/components/BaseLayerSwitcher/BaseLayerSwitcher.scss

```diff
@@ -0,0 +1,74 @@
+.rs-base-layer-switcher {
+  position: relative;
+  height: 80px;
+  width: 115px;
+  transition: 250ms width;
+  overflow: hidden;
+
+  &.open {
+    width: 360px;
+  }
+
+  .rs-base-layer-switch-layers {
+    display: flex;
+    overflow: auto;
+    height: 100%;
+    width: 360px;
+
+    .rs-base-layer-switch-button {
+      position: relative;
+      height: 85%;
+      min-width: 100px;
+      margin: 4px;
+      overflow: hidden;
+      border: 2px solid white;
+      background-color: gray;
+
+      &.open {
+        &:hover {
+          border: 2px solid red;
+        }
+
+        &.rs-active {
+          border: 2px solid red;
+        }
+      }
+
+      .rs-base-layer-switch-title {
+        position: absolute;
+        width: 100%;
+        min-height: 11px;
+        font-size: 9px;
+        padding: 2px 0;
+        background-color: rgba(255, 255, 255, 0.9);
+        text-align: center;
+        z-index: 2;
+
+        &.closed {
+          color: white;
+          background-color: rgba(0, 0, 0, 0.7);
+        }
+      }
+
+      .rs-base-layer-switch-image {
+        vertical-align: center;
+        width: 100px;
+        height: 100px;
+      }
+
+      img[alt] {
+        color: gray;
+      }
+    }
+
+    .rs-base-layer-switch-close {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      height: 85%;
+      margin: 4px;
+      background-color: white;
+      border: 2px solid white;
```

Comment:
this class a common css with the other buttons, move them at the same place.

Reply:
done